### PR TITLE
Limit number of comments shown

### PR DESCRIPTION
**Pull request by shulijones**
https://shulijones-step-2020.appspot.com/
Allow the user to select the maximum number of comments to be shown. 

Does not yet support error handling; has old timestamp features, so will need to be hand-merged after approval.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -35,20 +35,25 @@
...
+/** Servlet that handles guestbook comments */
+@WebServlet("/guestbook")
 public class DataServlet extends HttpServlet {
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     ArrayList<Comment> comments = new ArrayList<Comment>();
+    final int maxComments = Integer.parseInt(
+      request.getParameter("maxcomments"));
```
Comment (kyracq) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: I think it's standard to separate words with dashes for HTML

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -45,6 +45,13 @@ <h3> Sign my guest book! </h3>
       </form>
       <br>
       <h3>Past entries</h3>
+      <label for:"comments-num">How many entries?</label>
```
Comment (kyracq) [inline portfolio/src/main/webapp/index.html]: Colon after "for" could be an equals sign for consistency

**Review by kyracq — APPROVED** [review]
LGTM, minor formatting suggestions

**Review by otobin — APPROVED** [review]
LGTM!

**Pull request merged**